hedera-node/src/test/java/com/hedera/services/sigs/metadata/lookups/HfsSigMetaLookupTest.java

```diff
@@ -1,114 +0,0 @@
-package com.hedera.services.sigs.metadata.lookups;
-
-/*-
- * ‌
- * Hedera Services Node
```

Comment:
This test class is deleted accidentally ?

Reply:
Removed intentionally, as `HfsSigMetaLookUp` will also be removed in future PRs after removing dependency in `SigRequirementsTest`